changelog/25953.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:improvement
```

Comment:
```suggestion
```release-note:change
```

Reply:
Since 403s won't be exactly the same, I think this is worthwhile to call out as a change, in case anybody was doing any specific handling (for example, comparing against the exact error message).